libs/util/types/src/lib/metadata/metadata.model.ts

```diff
@@ -131,8 +139,8 @@ export interface DatasetRecord extends BaseRecord {
   datasetUpdated?: Date
   lineage: string // Explanation of the origin of this record (e.g: how, why)"
   distributions: Array<DatasetDistribution>
-  spatialExtents: Array<DatasetSpatialExtent> // not handled yet
-  temporalExtents: Array<DatasetTemporalExtent> // not handled yet
+  spatialExtents?: Array<DatasetSpatialExtent> // not handled yet
+  temporalExtents?: Array<DatasetTemporalExtent> // not handled yet
```

Comment:
Having less optional keys as possible was intended, since it means we can always expect objects to have roughly the same form; since extents are arrays, empty arrays can be used if no extent is available